accountsplus/settings.py

```diff
@@ -0,0 +1,70 @@
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.apps import apps
+
+
+def get_setting(setting_str, default_value=None):
+    try:
+        return getattr(settings, setting_str, default_value)
```

Comment:
@nasief You may want to add another parameter to `get_setting` method to check if this setting is required or not, and pass a default value only if the setting you are trying to get is optional.